app/services/finance/ecf/mentor/statement_calculator.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    module Mentor
+      class StatementCalculator
+        class << self
+          def event_types
+            %i[
+              started
+              completed
+            ]
+          end
+
+          def event_types_for_display
+            %i[
+              started
+              completed
+            ]
+          end
+        end
+
+        attr_reader :statement
+
+        def initialize(statement:)
+          @statement = statement
+        end
+
+        def vat
```

Comment:
`total_vat` so its not confusable with `vat_rate` or total including vat?

Reply:
This is just the calculated `vat` figure..

Reply:
Sure, I just meant the method name is maybe a bit ambiguous; if I came across something like this I would need to look at the implementation of `vat` to know if this is the vat rate the total vat amount:

```
<%= @statement_calculator.vat %>
```

Maybe its just how it is in my head, but if I read VAT I assume/see 20% 🙂